airbyte-integrations/connectors/source-google-ads/source_google_ads/streams.py

```diff
@@ -35,16 +64,21 @@ def chunk_date_range(
 
     # As in to return some state when state in abnormal
     if start_date > end_date:
-        return [{field: start_date.to_date_string()}]
+        start, end = get_date_params(start_date.to_date_string(), time_zone=time_zone, range_days=range_days)
+        return [{"start_date": start, "end_date": end}]
```

Comment:
I believe, we can avoid duplicating code by setting dates.

Reply:
@vitaliizazmic Done. I refactored this line. thanks for this suggestion!